front/app/containers/DisabledAccount/messages.ts

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  title: {
+    id: 'app.containers.DisabledAccount.title',
+    defaultMessage: 'Your account has been temporarily disabled',
+  },
+  text: {
+    id: 'app.containers.DisabledAccount.text1',
+    defaultMessage:
+      'Your account on {organizationName} Platform has been temporarily disabled for a violation of the community guideline. For more information on this, you can consult the {TermsAndConditions}.',
+  },
+  termsAndConditions: {
+    id: 'app.containers.DisabledAccount.termsAndConditions',
+    defaultMessage: 'terms & conditions',
+  },
```

Comment:
Why did you put this in a new string? :-) it can be in the original string, right?

Reply:
because this needed to be inside a link.

Reply:
I missed that.